src/winetricks

```diff
@@ -6475,6 +6475,34 @@ load_dxsdk_jun2010()
 
 #----------------------------------------------------------------
 
+w_metadata dxvk dlls \
+    title="Vulkan-based D3D11 implementation for Linux / Wine" \
+    publisher="Philip Rebohle" \
+    year="2018" \
+    media="download" \
+    file1="dxvk-0.54.tar.gz" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d11.dll" \
+    installed_file2="$W_SYSTEM32_DLLS_WIN/dxgi.dll"
+
+load_dxvk()
+{
+    _W_dxvk_version="$(echo "$file1" | sed 's/^dxvk\-\|.tar.gz$//g')"
+
+    # https://github.com/doitsujin/dxvk
+    w_download "https://github.com/doitsujin/dxvk/releases/download/v0.54/dxvk-0.54.tar.gz" 1c2f186baaa01d2de7b832f6f05021bdd29eccb65fc197c8b15adfd4e08f9640
+    w_try_7z "$W_TMP" "$W_CACHE/$W_PACKAGE/dxvk-$_W_dxvk_version.tar.gz" -y
+    w_try_7z "$W_TMP" "$W_TMP/dxvk-$_W_dxvk_version.tar" -y
```

Comment:
Is there a reason you're using 7z to extract a tar? I mean, it can, but every Unix has tar available.

Reply:
Yes... I would much rather have used **tar**... But as **winetricks** has to run on **BSD**, **MacOS X**, **Cygwin** I don't want to step into the minefield of incompatible **tar** options!

Reply:
Extracting a gzip file is standard across all platforms
-z gzip
-x extract
-f file (to extract)

See, e.g., the OpenBSD manpage:
https://man.openbsd.org/tar

It's used in several places already:
```
austin@austin2:~/src/winetricks$ git grep "tar.*zxf"
src/winetricks:    w_try tar -zxf "$W_CACHE/$W_PACKAGE/$file1" baekmuk-ttf-2.2/ttf
src/winetricks:    w_try tar -zxf "$W_CACHE/$W_PACKAGE/$file1" "${file1%.tar.gz}/fonts"
src/winetricks:    w_try tar -zxf "$W_CACHE/$W_PACKAGE/$file1"
src/winetricks:    w_try tar -zxf "$W_CACHE/$W_PACKAGE/$file1"
src/winetricks:        tar -zxf "$W_CACHE/qq/QQ.tar.gz"
```

It's only when you get into advanced options that you lose portability. The GNU page mentions which options are GNU specific (mostly long options).

Besides, I don't think Vulkan works on Cygwin ;)